Gopkg.toml

```diff
@@ -32,10 +24,6 @@
   revision = "0c8571ac0ce161a5feb57375a9cdf148c98c0f70"
 
 [[constraint]]
-  name = "github.com/moby/buildkit"
```

Comment:
looks like a bad merge resolution here, you'll have to add these back in?

Reply:
Ugh, yeah somehow this is hitting dep issues even though it builds locally for me. Working on it

Reply:
FYI these should be resolved now. I think we're fine removing these and letting dep figure resolve the correct version based on transitive dependencies